src/routes/github/webhook/webhook-receiver-post.ts

```diff
@@ -31,10 +31,21 @@ export const WebhookReceiverPost = async (request: Request, response: Response):
 	});
 	logger.info("Webhook received");
 	try {
-		const { webhookSecret, gitHubServerApp } = await getWebhookSecret(uuid);
-		const verification = createHash(request.rawBody, webhookSecret);
+		const { webhookSecrets, gitHubServerApp } = await getWebhookSecrets(uuid);
+		const isVerified = webhookSecrets.some((secret, index) => {
+			const matchesSignature = createHash(request.rawBody, secret) === signatureSHA256;
+			/**
+			 * The latest updated webhook secret will be at index 0,
+			 * Once we stop receiving logs with index other than 0,
+			 * can then completely remove the old webhook secrets.
+			 */
+			if (matchesSignature) {
+				logger.info("Matched with the webhook of index: ", index);
```

Comment:
If it matches with secret's index other than 0, add logging in that case only. Otherwise it will clutter the logs.

Reply:
I think its ok to log all indexes for the webook secrets cause it gives us an idea of which webhook secret are in use, otherwise there is no way to figure it out. And once we figure out which ones are ont in use, we can then proceed to remove it forever.

Reply:
Can we update this to follow the logging pattern we use: object + message e.g. `logger.info({ index }, "Matched webhook index");`